spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncherProperties.java

```diff
@@ -43,4 +71,16 @@ public void setModules(String[] modules) {
 		return modules;
 	}
 
+	@PostConstruct
+	public void deduceModuleArguments() throws Exception {
```

Comment:
Actually, what if boot could do what we want?
What about mapping the equivalent of this yaml form:
```yaml
modules
  foo:    // module "label"
    coordinates: org.springframework.cloud.stream.module:time-source:1.0.0.BUILD-SNAPSHOT
    arguments:
      foo: bar
      quizz: fix
```